### ISSUE TITLE
Concept: Revisit "War in the South" Narrative

### ISSUE THREAD

**Issue by Remerod** [CK3RealmsinExile/RealmsInExile#291]
So VectorMaximus wrote us a narrative of what happened in the South during the War of the Ring. Which was meant to be translated to the mods game/Story narrative down there. While some parts of it got into the mod other remain to be implemented. While the game is very playable and focus moves towards 2.3 and 3.0 and forward, i thought it important to write these up so they are not forgotten, because their inclusion would very much enrich the gameplay in 2.2. 

**The war in the South** – CANONICAL route
<details>
<summary>Click to expand</summary>

**3000 T.A. – Melee a Trois.**
In the bay of Tulwang, an uneasy peace endures.
• The Solar Dynasty in Ray, originally fervent Vatrans, have tempered their faith and ambitions with long trade with Bellakar. The current Solar Kralyi is a middling man who would seek peace over war. However, a passionate new speaker has begun to gather followers in the Dar, the eventual Serpent-Dynast, Kalach Hûl, originally of Abrakhan.
• Bellakar has lost Felaya, but it still controls its heartland, and maintains its strength. The current king, Nîlûhîn II, seeks to reclaim the lands of his kingdom lost, but is wary of a potential war on two fronts.
• The Thon-an-Kharlok, the Storm-Host, readies across the bay of Tulwang. However, in secret, some of the Anballukhori lords speak of perhaps breaking the yoke of the Storm-marshals. The current king of An-Balkumagan is weak and indecisive, but his brother, well respected by the bureaucrats and army, chafes under submission to a warlord and a king that has not been seen in the kingdom for many centuries. He has the ear of his nephew and would seek to forge a new path for An-Balkumagan, free of the chains which have bound them to the dark lord, proud in their heritage as Adûnaim. He would even see peace with Bellakar, their distant kin – for the right concessions.

**3007 T.A - Tedjin civil war.**
• Conservative darkness-aligned, clan-based Tedjin in the north and the great dune sea launch a revolt against the reforms of the newly ascended Kataj (High Chieftain), which aims to settle the Tedjin and integrate them at long last with their neighbors. The revolt is also religion based, as the settled Tedjin of the south – derrogatedly referred to as the Tedyîn by their northern kin, symbolizing their absorption of Bellakari traditions and religion – have increasingly adopted the Batan an-Izindi, abandoning the profane and dark rites of their ancestors. With support of Sauron via Umbar, the conservative clans win the war, and with their fervor stoked, prepare to at last restore the long dead Tedjin Katajar, which once dominated all of Bellakar from Umbar to Bur Esmer.

**3009 T.A. – Assault on Mardruak**
• The Tedjin have completed their preparations and launch a significant assault on the city-states of Mardruak. Only Narik-zadan maintains its independence, its ancient walls holding off the forces of the Tedjin, its fleet able to keep supplies flowing into the great city. The walls of Nykkea are torn down and the city is devastated. Attempts to push south are met with fierce Felayan resistance, with the Army of the Righteous reformed by the Izindubeths to counteract the invasion. Bellakar sends forces north to aid the Felayans, and the First Lord bows to the crown of Bellakar once more, in exchange for privileges and autonomy.

**3011 T.A. – Dispute in the Dar**
• The Solar Dynasty is desposed, its lord and many of the family murdered in the night, with the Serpent-Lord Mardat proclaimed the new Kralyi of Ray. Unbeknownst to most in the Dar, the Serpent-Lord only succeeded by the personal aid of the Golden King, who came in person with his dread forces to ensure the coup’s success. With the aid of the Golden King, the new master of the Ray begins a brutal purge of the Dar and the surrounding countryside, enforcing the cult of Khaz-gramaze by the sword and inquisition.

**3012 T.A. – Fall of Narik-zadan**
• The Tedjin at last take the great city, their conquest of Mardruak complete. They double their assault on Felaya, gold continuing to be supplied by foreign powers, primarily Abrakhan.

**3014 T.A. – Assault on Bellakar**
• Minor skirmishing between the Dar and Bellakar had been happening ever since the ascension of the Serpent-Lord, but now it explodes into full-blown war. To the south, the Storm-Marshal is at last given the order to cross the bay and assault Bellakar. Esmer and Korlea are laid to siege, Esmer falling in 3015. With abhorrent forces pouring in from the east via Abrakhan, defeat seems inevitable.
• Unable to yet dislodge the Falayans from the monastery of Urud an-Khibil, the Tedjin manage to take the city of Hazaj Tollin, cutting the Felayan forces in half. This breakthrough is made possible when Bellakari forces are suddenly recalled to face the southern threat, the Tedjin left to the regional Felayan forces

**3016 T.A. – The Surprise Peace**
• However, the Storm-host has overplayed its hand. The King of An-Balkumagan dies, and his son, still fairly young, leans on his uncle (the viceroy of Tulwang) for counsel. Long discontent with their subjugation to a lower dynasty of warlords, they plot to at last break free of the Storm-host. An-Balkumagan rises in Revolt, supported by Etulia and Tulwang. Simultaneously, the Tulwangi see this as a chance to finally rise up against the Anbalukkhori. The Storm-host is abruptly forced to recall all forces to deal with this internal dispute. With the siege of Korlea lifted by Nîlûhîn, the Ray’s advance up towards Nilulonde is halted.

**3019 T.A. – The Black Tower Falls**
• With the fall of Sauron, Abrakhan – governed by the Golden King’s lich lieutenants, begins to collapse. Without the resources of Abrakhan (such as orcs, gold, and sorcerers), the Serpent-Dynasty can no longer maintain its grip upon Ray or the assaults on Bellakar. Revolts from loyalists of the Solar Dynasty and religious dissidents centered in Tresti results in a near total collapse in the Dar. The Serpent-Lord is found crucified in the great square, the phrase ‘The Gods Judgement’ branded on his skin for his crimes. Meanwhile, the Storm-marshal falls in combat before his stronghold of Khorsaj, the fortress sacked by the Tulwangi and Anbalukkhori. The two revolts, previously tenuously allied, quickly turn on each other.
• Some of the Tedjin lords assault the southern provinces of Umbar, taking some of them before meeting concerted resistance as they approach the haven proper. Gold stops flowing from Sauron and his dominions, and the Tedjin grow restless at their stalled wars.

**F.A. 2 – The Accords of Adûnaphazân**
• Nîlûhîn II meets with the rebel alliance of the Ray which overthrew the Serpent Dynasty and manages to broker a peace that both sides can accept. The Council of Regents, defunct since the Vatran coup, is restored in power to the Dar. Meanwhile, across the bay, An-Balkumagan is forced to yet again accept their losses in northern Tulwang, focusing on consolidating their grip on their heartland and Etulia. Bellakar now turns north to finally reinforce the Felayans and try to push back the Tedjin.

</details>

So this Storyboard basically describe the "canon" route for the Far harad area, but each row/point need's to be fleshed out with alternative routes, just as the Tedjin civil war, you have a victorious site for both the "canon" & "not-canon" sides, and how these alternative routes leads into the future of the whole needs to be fleshed out in adopting them into the mods story narrative.

**What's left to do:**
So while some of these are already done, the majority is not. It would be preferable if someone wants to pic some off this up to start on the earlier rows/points. But basically these rows/points below need to be tackled

•  **3000 T.A. – Melee a Trois.**
-Bellakar seems to be the one whom most fits in in this narrative.
-Ray, could be worked on here to fit more into a **Dispute in the Dar** narrative
-The Thon-an-Kharlok, basically exist as a base game setup without narrative, would need to be worked on. 

• **3007 T.A - Tedjin civil war.**
-Basically done

• **3009 T.A. – Assault on Mardruak**
-Basically done, is worked into the Tedjin civil war as a mechanic, could maybe use some narrative events. 

• **3011 T.A. – Dispute in the Dar**
-Not Started.

• **3012 T.A. – Fall of Narik-zadan**
-Basically exist as a mechanic, does not have story narrative event with Abrakhan. 

• **3014 T.A. – Assault on Bellakar**
-Not Started.

• **3016 T.A. – The Surprise Peace**
-Not Started

• **3019 T.A. – The Black Tower Falls**
-Not Started, but realistically this can't be finished before 5.0

• **F.A. 2 – The Accords of Adûnaphazân**
-Not Started, could maybe be finished before **3019 T.A. – The Black Tower Falls** but the other story/narrative events would have need to be finished.

**Comment by VectorMaximus** [comment]
Anybody wishing to discuss possibly taking this up and wanting additional input on my overall thoughts to the progression of the various theaters, feel free to DM me on Discord.

**Comment by Remerod** [comment]
Since we started doing a "struggle" for the bay of tulwang, I'll move this to the non-critical stuff

**Comment by Jaco-Daan** [comment]
Just commenting here to remind myself to come look at this when we start with 4.0

**Comment by Jaco-Daan** [comment]
![Rumbl_in_Ray](https://user-images.githubusercontent.com/62937916/236928799-c01c1f3f-d6be-4ac4-af08-56f076033331.png)
The official plans for Ray, sponsored by Remerod

**Comment by Jaco-Daan** [comment]
So good progress has been made on this, only the SH event chain to test if I remember correctly.

**Comment by Jaco-Daan** [comment]
Closing this as we've finished this, and issues really aren't meant for suggestions

**Comment by Trigparmandil** [comment]
I'd point out that this is partially complete, while we have Stomhost content, we don't have the main storyline moving for Ray yet.
